astropy/utils/tests/test_data.py

```diff
@@ -1755,3 +1780,61 @@ def test_download_parallel_respects_pkgname(temp_cache, valid_urls):
                                pkgname=a)
     assert not get_cached_urls()
     assert len(get_cached_urls(pkgname=a)) == FEW
+
+
+@pytest.mark.parametrize("b", _shelve_possible_backends)
+def test_cache_with_different_shelve_backends(temp_cache, b, valid_urls):
```

Comment:
Just to be consistent with the way "b" is defined in the other test function above.
```suggestion
def test_cache_with_different_shelve_backends(b, temp_cache, valid_urls):
```